src/domain/SchemaValidation.ts

```diff
@@ -72,3 +80,19 @@ function validateUInt32(value: number): boolean {
 function validateUInt64(value: number): boolean {
   return Number.isInteger(value) && value >= ZERO && BigInt(value) <= MAX_UINT64;
 }
+
+function validateInt8(value: number): boolean {
+  return Number.isInteger(value) && value >= -MAX_INT8 && value <= MAX_INT8;
+}
+
+function validateInt16(value: number): boolean {
+  return Number.isInteger(value) && value >= -MAX_INT16 && value <= MAX_INT16;
+}
+
+function validateInt32(value: number): boolean {
+  return Number.isInteger(value) && value >= -MAX_INT32 && value <= MAX_INT32;
+}
+
+function validateInt64(value: number): boolean {
+  return Number.isInteger(value) && value >= -MAX_INT64 && BigInt(value) <= MAX_INT64;
```

Comment:
Is this validation working correctly for the negative part of it? [According to the BigInt docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/BigInt#operators), you can mix different numerical types for boolean operators, but to me, it seems to be error-prone. Just in case, it'd be better to use `BigInt(value) >= -MAX_INT64`.

Reply:
Done, thanks!